framework/db/src/main/java/com/cloud/utils/db/TransactionLegacy.java

```diff
@@ -1032,11 +1033,7 @@ public static void initDataSource(Properties dbProps) {
             final long cloudMaxWait = Long.parseLong(dbProps.getProperty("db.cloud.maxWait"));
             final String cloudUsername = dbProps.getProperty("db.cloud.username");
             final String cloudPassword = dbProps.getProperty("db.cloud.password");
-            final String cloudHost = dbProps.getProperty("db.cloud.host");
             final String cloudDriver = dbProps.getProperty("db.cloud.driver");
```

Comment:
If we inform the full URI, we can retrieve the driver from it, making this property unnecessary in this scenario.